### DISCUSSION TITLE
Exit Code 1 error preventing downloads

### DISCUSSION THREAD

**Discussion by Cronus82** [aclap-dev/video-downloadhelper#334] [Help and Q&A]
### Your question

Every video I try to download gives me this error:
Exit code: 1

moz-extension://705ff829-5e78-41fd-a614-cfef28388248/background/main.js:1
receive@moz-extension://705ff829-5e78-41fd-a614-cfef28388248/background/main.js:1:13157
nd</callCatchAppNotFound/</<@moz-extension://705ff829-5e78-41fd-a614-cfef28388248/background/main.js:1:42626

Has anyone else gotten this error and is there a fix for it?

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): daef24c82005491d
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "i686",
    "node": "10"
  },
  "home": "C:\\Users\\Brian",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
Some users have had that problem, it turned out in some cases the anti virus & firewall was the issue. 
Have you verified that the companion app isn't blocked / quarantined by anti virus & firewall.

**Reply by Fairgan** [reply]
for me, it was the bitfender firewall that prevented the ffprobe.exe to run.

**Reply by Cronus82** [reply]
Thanks, you were spot on. It seems Kaspersky was the problem. I disabled it and now I'm back in business!
